src/cppmangle.d

```diff
@@ -1467,7 +1467,13 @@ else static if (TARGET_WINDOS)
             buf.writeByte('?');
             mangleIdent(d);
             assert((d.storage_class & STCfield) || !d.needThis());
-            if (d.parent && d.parent.isModule()) // static member
+            Dsymbol parent = d.toParent();
+            // skip the enclosing 'export' scopes
```

Comment:
`'export' scopes` = C++ namespaces. I don't think the comment is needed though, the code is self-explaining IMO.